composer.json

```diff
@@ -12,6 +12,7 @@
         "barryvdh/laravel-translation-manager": "^0.5.1",
         "doctrine/dbal": "^2.7",
         "fideloper/proxy": "^4.0",
+        "fzaninotto/faker": "1.9.1",
```

Comment:
Interesting that this has moved from require-dev, wonder why a faker is needed for production code and not just in tests.

Reply:
I had some weird issues where this kept not installing properly.  I decided to be lazy and let webpack tree-shaking handle it.

Reply:
Can dig further if you want.